ibis/pandas/execution/tests/test_window.py

```diff
@@ -472,6 +453,31 @@ def test_window_with_preceding_expr():
     tm.assert_series_equal(result, expected)
 
 
+def test_window_with_mlb():
+    index = pd.date_range('20170501', '20170507')
+    data = np.random.randn(len(index), 3)
+    df = pd.DataFrame(data, columns=list('abc'), index=index)\
```

Comment:
You should be able to run `black` on this file and it will do the right thing. If you've installed `pre-commit` this should happen automatically whenever you make a commit.

Reply:
Interesting, `black` does run on my commits but it passed on this one.

Reply:
You can always avoid a `\` by wrapping the entire expression in parentheses if necessary.

Reply:
Hm, okay when you run `pre-commit run --files ibis/pandas/execution/tests/test_window.py` does that change the file?

Reply:
When I run the above it still says 'Passed' for black (and all other pre commit hooks)

Reply:
I wrapped the offending lines in parentheses to avoid the `\` in the next commit